bee-common/bee-common/src/logger/config.rs

```diff
@@ -57,6 +57,13 @@ impl LoggerOutputConfigBuilder {
         self
     }
 
+    /// Sets a collection of exclusions of a logger output.
+    /// A message is logged only if one of the exclusions is *not* part of the log's metadata target.
+    pub fn target_exclusions(mut self, target_exclusions: &[&str]) -> Self {
+        self.target_exclusions = Some(target_exclusions.iter().map(|f| f.to_string()).collect::<Vec<String>>());
```

Comment:
I think you should avoid the closure (clippy will complain about it iirc)

Reply:
Ah, I didn't run clippy on bee-common! Good spot.